### ISSUE TITLE
Websocket responds to a ping with a faulty packet

### ISSUE THREAD

**Issue by ticaki** [hyperion-project/hyperion.ng#1821]
- [x] I confirm that this is an issue rather than a question.

I have previously searched the web, chatgpt and here for relevant information, but only found error analyses for the server.

## Bug report
When I send a ping to the server via a web socket connection, I receive an incorrect response that closes the connection.
I currently use NodeJS and the `ws` package for this.

#### Steps to reproduce

Send a ping to hyperion via websocket.

#### What is expected?
a flawless pong :)

#### What is actually happening?
My websocket connection closes and I have to go alternative ways to get a live check.

Log of hyperion:

```
2024-12-30T08:22:56.755Z [WEBSOCKET] (DEBUG) (WebSocketClient.cpp:30:WebSocketClient()) New connection from ::ffff:192.168.178.65
2024-12-30T08:22:56.782Z [WEBSOCKET] (DEBUG) (WebSocketClient.cpp:213:sendClose()) Send close to ::ffff:192.168.178.65: 1000 
```

Nodejs Error:
```
2024-12-30 09:22:56.769  - error: hyperion-ng2.0 (1969) [Error:] Invalid WebSocket frame: FIN must be set
```

#### System

Hyperion Server:
- Build:             (HEAD detached at 2.0.16) (Paulchen-Panther-cb85d2d/a93d79b-1705568419)
- Build time:        Jul 29 2024 19:40:14
- Git Remote:        https://github.com/hyperion-project/hyperion.ng
- Version:           2.0.16
- UI Lang:           de (BrowserLang: de-DE)
- UI Access:         advanced
- Avail Screen Cap.: dispmanx,framebuffer,qt
- Avail Video  Cap.: v4l2
- Avail Audio  Cap.: audio
- Avail Services:    boblight,cec,effectengine,forwarder,flatbuffer,protobuffer,mDNS,SSDP,borderdetection
- Config path:       /home/tim/.hyperion
- Database:          read/write
- Mode:              Non-GUI

Hyperion Server OS:
- Distribution:      Debian GNU/Linux 12 (bookworm)
- Architecture:      arm64
- CPU Type:          Raspberry Pi 3 Model B Rev 1.2
- CPU Revision:      a02082
- Kernel:            linux (6.6.62+rpt-rpi-v8 (WS: 64))
- Root/Admin:        false
- Qt Version:        6.4.2
- Python Version:    3.11.2
- Browser:           Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.2 Safari/605.1.15

**Comment by Lord-Grey** [comment]
@ticaki Thanks for sharing your finding. It is know issue see #1816 and was already fixed by #1819.

Would be great, if you could install the nightly build , retest and share your findings.

**Comment by ticaki** [comment]
@Lord-Grey 
```
2024-12-30 22:51:21.919  - debug: hyperion-ng2.0 (17472) [Hyperion (192.168.178.90)] Ping
2024-12-30 22:51:21.920  - debug: hyperion-ng2.0 (17472) [Hyperion (192.168.178.90)] Pong received
2024-12-30 22:51:21.921  - debug: hyperion-ng2.0 (17472) [Hyperion (192.168.178.90)] Reset alive check
```
fixed!
Thanks
